config.go

```diff
@@ -4,9 +4,14 @@ import (
 	"io/ioutil"
 	"log"
 
-	yaml "gopkg.in/yaml.v2"
+	yaml "github.com/ghodss/yaml"
```

Comment:
this dependency was introduced in the previous PR. it fixed the problem of the go-yaml library excluding certain items like the operationId in a path/operation.